src/ThermalScattering.tex

```diff
@@ -1,49 +1,80 @@
+  \item \textbf{\Block{ITCA}} and \textbf{\Block{ITCAI}}---contains angular distributions for elastic scattering.
+                                The \Block{ITCA} exists if the material has coherent and/or incoherent elastic scattering
+                                (\nxs{5}$\neq0$, \jxs{6}$\neq0$ and \nxs{6}$\neq-1$). The \Block{ITCEI} only exists for mixed mode
+                                elastic scattering (\nxs{5}$=5$, \jxs{9}$\neq0$ and \nxs{6}$\neq-1$). See \Sectionref{sec:ITCABlock}.
 \end{enumerate}
 
 \subsection{\NXS\ Array}\label{sec:NXSThermalScattering}
 
 \begin{NXSTable}{thermal scattering}
-  1  & ---         & Length of second block of data (\XSS\ array)               \\
-  2  & \var{IDPNI} & Inelastic scattering mode                                  \\
-  3  & \var{NIL}   & Inelastic dimensioning parameter                           \\
-  4  & \var{NIEB}  & Number of inelastic exiting energies                       \\
-  5  & \var{IDPNC} & Elastic scattering mode (coherent=4, incoherent$\ne$4)     \\
-  6  & \var{NCL}   & Elastic dimensioning parameter                             \\
-  7  & \var{IFENG} & Secondary energy mode (discrete=0, skewed=1, continuous=2) \\
+  1  & ---         & Length of second block of data (\XSS\ array)                \\
+  2  & \var{IDPNI} & Inelastic scattering mode                                   \\
+  3  & \var{NIL}   & Inelastic dimensioning parameter                            \\
+  4  & \var{NIEB}  & Number of inelastic exiting energies                        \\
+  5  & \var{IDPNC} & Elastic scattering mode (no elastic data=0, incoherent=3, coherent=4, mixed=5) \\
+  6  & \var{NCL}   & Elastic dimensioning parameter for the first elastic block  \\
+  7  & \var{IFENG} & Secondary energy mode (discrete=0, skewed=1, continuous=2)  \\
+  8  & \var{NCLI}  & Elastic dimensioning parameter for the second elastic block \\
      & \ldots      & \\
-  16 & ---         & 
+  16 & ---         &
   \label{tab:NXSThermalScattering}
 \end{NXSTable}
 
 \subsection{\JXS\ Array}\label{sec:JXSThermalScattering}
+
 \begin{JXSTable}{thermal scattering}
-  1  & \var{ITIE} & Inelastic energy table               \\
-  2  & \var{ITIX} & Inelastic cross sections             \\
-  3  & \var{ITXE} & Inelastic energy/angle distributions \\
-  4  & \var{ITCE} & Elastic energy table                 \\
-  5  & \var{ITCX} & Elastic cross sections               \\
-  6  & \var{ITCA} & Elastic angular distributions        \\
-     & \ldots     & \\
-  32 & ---        &
+  1  & \var{ITIE}  & Inelastic energy table               \\
+  2  & \var{ITIX}  & Inelastic cross sections             \\
+  3  & \var{ITXE}  & Inelastic energy/angle distributions \\
+  4  & \var{ITCE}  & Elastic energy table (used for coherent elastic scattering if \\
+     &             & \nxs{5}=4 or 5, and used for incoherent elastic scattering if \nxs{5}=3)                 \\
```

Comment:
So I think this is a little confusing the way it's presented. It might be better if the extra comments were indicated with table footnotes. I can push something up to this branch or another branch if desired.

Reply:
This table is in a custom table environment. I have tried to get it to work with footnotes but did not manage to do so. Since I don't want to waste any more time, I'm leaving this table and the text as is.